sale_analytic_account/views/product_template_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--add create_analytic_account field into form view-->
+    <record id="product_template_create_analytic_account_view_form" model="ir.ui.view">
+        <field name="name">product.template.create.analytic.account.form.inherit</field>
+        <field name="model">product.template</field>
+        <field name="priority" eval="90"/>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="arch" type="xml">
+            <field name='type' position="after">
+                <field name="create_analytic_account"/>
+            </field>
+        </field>
+    </record>
+
+    <!--add create_analytic_account into search view-->
+    <record id="product_template_create_analytic_account_view_search" model="ir.ui.view">
+        <field name="name">product.template.create.analytic.account.search.inherit</field>
+        <field name="model">product.template</field>
+        <field name="priority" eval="90"/>
+        <field name="inherit_id" ref="product.product_template_search_view"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <filter name="create_analytic_account" string="Create Analytic Account" domain="[('create_analytic_account', '=', True)]"/>
+            </field>
+            <xpath expr="//group[1]" position="inside">
+                 <filter string="Create Analytic Account" name="create_analytic_account" context="{'group_by':'create_analytic_account'}"/>
+            </xpath>
+        </field>
+    </record>
```

Comment:
```suggestion
```

It's not necessary to add this to the search view, it's just a configuration field